addons/purchase_stock/views/purchase_views.xml

```diff
@@ -24,6 +24,13 @@
             <xpath expr="//field[@name='order_line']/tree//field[@name='date_planned']" position="after">
                 <field name="move_dest_ids" invisible="1"/>
             </xpath>
+            <xpath expr="//div[@name='date_planned_div']" position="replace">
```

Comment:
Just some questions about this:
- The field date_planned lost his readonly condition, it is correct ?
- Maybe there is a simple way to avoid the replace ?

Reply:
Not really, it just restricts future xpath about these specific fields in others module. In my poiint of view, you can keep with a replace.

Reply:
Ah right, good point. Yeah I unfortunately don't see a good way around this for now because there's a "only show 2 dates at a time" requirement that makes it tricky. Hopefully no future complications happen!